packages/vite-node/src/cli.ts

```diff
@@ -62,18 +64,24 @@ export interface CliOptions {
   root?: string
   config?: string
   watch?: boolean
+  'inline-deps'?: string | string[]
 }
 
 async function run(options: CliOptions = {}) {
   const files = options.files || options._ || []
+  const inlineDeps = options['inline-deps'] || []
   const server = await createServer({
     logLevel: 'error',
     configFile: options.config,
     root: options.root,
   })
   await server.pluginContainer.buildStart({})
 
-  const node = new ViteNodeServer(server)
+  const node = new ViteNodeServer(server, {
+    deps: {
+      inline: Array.isArray(inlineDeps) ? inlineDeps : [inlineDeps],
+    },
```

Comment:
Should we also port the rest of the relevant configuration as CLI flags? Or would you rather keep the bare minimum, and add stuff when needed?

Reply:
I think we should at least port `fallbackCJS` (with `no-` prefix for `false`) - I think `minimist` provides this functionality out of the box, but I am not sure.

Also, what about regexp?